src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -278,38 +277,30 @@ protected override TDeclarationNode AddNamespace<TDeclarationNode>(TDeclarationN
             CodeGenerationOptions options,
             CancellationToken cancellationToken)
         {
-            var memberDeclaration = destinationMember as MemberDeclarationSyntax;
-            if (memberDeclaration == null)
-            {
-                return destinationMember;
-            }
+            var s_generator = CSharpSyntaxGenerator.Instance;
+            var currentParameterList = s_generator.GetParameters(destinationMember);
```

Comment:
nit: probably rename the parameter to just `destination`, now that we support more than just members.